src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -293,8 +318,22 @@ class CollabWrapper extends PureComponent<Props, State> {
     }
   };
 
-  private handleRemoteSceneUpdate = (
+  private reconcileElements = (
```

Comment:
just a temp, not sure if we wanna keep it, but I need to get the elements so I split it from `handleRemoteSceneUpdate `